src/_zkapauthorizer/tests/test_model.py

```diff
@@ -1153,3 +1162,45 @@ def test_roundtrip(self, pass_):
             Pass.from_bytes(pass_.pass_bytes),
             Equals(pass_),
         )
+
+
+class ReplicationTests(TestCase):
+    """
+    Tests for replication and recovery.
+    """
+
+    def test_recover(self):
+        """
+        Given a snapshot returned by ``snapshot`` can be loaded into an empty
+        ``VoucherStore`` using ``VoucherStore.call_if_empty`` with
+        ``StatefulRecoverer.recover``.
+        """
+        store = self.useFixture(
+            ConfiglessMemoryVoucherStore(
+                # Time is not relevant to this test
+                datetime.now,
+            )
+        ).store
+        # XXX Use VoucherStore.snapshot once it is available
+        snapshot_bytes = b"".join(statements_to_snapshot(snapshot(store._connection)))
```

Comment:
Not really part of this PR I guess, but that reads weirdly .. maybe pointing to a different name for `snapshot()`? (e.g. "`statements()`" if it is read literally, probably, but that's not very descriptive.

E.g. `statements_to_snapshot(snapshot_statements(...))` maybe...?